spec/gameobj-data/boxes_and_picking_spec.rb

```diff
@@ -46,6 +33,48 @@
       %{weathered},
     ]
 
+    # Separating Boon boxes to make it easier to read
+    # Removing boon_box_prefixes as this was found to be inaccurate after closer review
+    #boon_box_prefixes = %w[austered gilded ornate crude brass-inlaid carved delicate red cracked deeply-scored] # Similar to Box_metals -> use as desc
```

Comment:
If you want to rename this, please delete it instead of leaving it commented out.